packages/utilities/src/warn.ts

```diff
@@ -0,0 +1,63 @@
+let _warningCallback = _warn;
+
+export interface IStringMap {
```

Comment:
Good news! I did this, and it caught some bugs where I misplaced some warnings! Yay typings!